pkg/operator/target_config_reconciler.go

```diff
@@ -1594,6 +1630,18 @@ func (c *TargetConfigReconciler) isKubeVirtDeployed() (bool, error) {
 	return false, nil
 }
 
+func (c *TargetConfigReconciler) isPSIenabled() (bool, error) {
+	_, err := os.Stat(c.psiPath)
```

Comment:
Even though the [operator deployment](https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/manifests/cluster-kube-descheduler-operator.clusterserviceversion.yaml#L347) does not currently tolerate control-plane/master taints it might be extended to do so in the future. When a PSI is enabled are non-worker nodes/container running on non-worker nodes expected to have `/proc/pressure` present too?

Reply:
PSI is supposed to be enabled via a {{machineConfig}} object. It only depends on the label selector set there by the cluster admin.